README.md

```diff
@@ -137,45 +137,47 @@ const result = await multiUserSession.prompt([
 
 Because of their special behavior of being preserved on context window overflow, system prompts cannot be provided this way.
 
-### Emulating tool use or function-calling via assistant-role prompts
+### Function Calling
```

Comment:
Let's stick with "tool use" instead of "function calling" throughout, since our API will use `tools`.